TypeCobol.Test/Parser/Programs/TypeCobol/ProcedureCall-ParamCompatibily.rdzSYM.txt

```diff
@@ -116,6 +118,22 @@ LocalStorageData:
     Picture: 9(10)
   Level: 1
   IsFiller: False
+  picNum10SyncLeft (.NET Type=VariableSymbol, Kind=Variable)
+  Flags: [LOCAL_STORAGE, Sync]
+  Owner: ProcedureCall-ParamCompatibily
+  Type:
+    (.NET Type=PictureType, Tag=Picture)
+    Picture: 9(10)
+  Level: 1
+  IsFiller: False
+  picNum10SyncRight (.NET Type=VariableSymbol, Kind=Variable)
+  Flags: [LOCAL_STORAGE, Sync]
```

Comment:
@smedilol 
Should we store the information regarding Left/Right for Sync flag if specified?

Reply:
Yes we need this information to check our tests.

Reply:
So after discussing with @sarterbe, he confirmed that `LEFT` or `RIGHT` has no impact. It is treated merely as documentation by the compiler but does not affect the runtime. Therefore it has no semantics.

Reply:
Ok, so it's ok with only `Sync`